source/features/search-releases.tsx

```diff
@@ -0,0 +1,267 @@
+		return Promise.reject(error);
+	}
+};
+
+const fetchReleasesV3 = async (ownerName, repoName): Promise<Release[]> => {
+	try {
+		const jsonResponse = await api.v3(`repos/${ownerName}/${repoName}/releases?per_page=100`) as ReleasesResponseV3[];
+
+		const releases: Release[] = jsonResponse.map(item => {
+			return {tag: item.tag_name, htmlPage: item.html_url};
+		});
+
+		return releases;
+	} catch (error) {
+		return Promise.reject(error);
+	}
+};
+
+const addSelectMenu = (releases: Release[]) => {
+	const container = select('.subnav');
+
+	const child = (
+		<div class="float-right">
+			{getSelectMenu({
+				title: 'Releases',
+				children: getAllReleasesLink(releases)
+			})}
+		</div>
+	);
+
+	container.append(child);
+};
+
+const getAllReleasesLink = (releases: Release[]) => {
+	const selectedTag = getSelectedTagFromUrl();
+
+	return releases.map(release => {
+		return (
+			<a
+				href={release.htmlPage}
+				class={`select-menu-item js-navigation-item ${
+					release.tag === selectedTag ? 'selected' : ''
+				}`}
+			>
+				{checkInline()}
+				<div class="select-menu-item-text js-select-button-text">
+					{release.tag}
+				</div>
+			</a>
+		);
+	});
+};
+
+const getSelectMenu = ({title, children}) => {
+	const selectedTag = getSelectedTagFromUrl();
+
+	// https://styleguide.github.com/primer/components/select-menu/#filter
+	return (
+		<div class="refined-github-search-releases select-menu js-menu-container js-select-menu">
+			<button
+				class="btn select-menu-button js-menu-target"
+				type="button"
+				aria-haspopup="true"
+				aria-expanded="false"
+			>
+				<i>Filter:</i>&nbsp;
+				<span class="js-select-button">
+					{selectedTag ? selectedTag : title}
+				</span>
+				&nbsp;
+			</button>
+			<div class="select-menu-modal-holder js-menu-content js-navigation-container">
+				<div class="select-menu-modal">
+					<div class="select-menu-header" tabindex="-1">
+						<button class="btn-link close-button js-menu-close" type="button">
+							{closeIcon()}
+						</button>
+						<span class="select-menu-title">{title}</span>
+					</div>
+					<div class="js-select-menu-deferred-content">
+						<div class="select-menu-filters">
+							<div class="select-menu-text-filter">
+								<input
+									type="text"
+									id="filter-field"
+									class="form-control js-filterable-field js-navigation-enable"
+									placeholder="Type to filter..."
+									aria-label="Type to filter"
+									autocomplete="off"
+								/>
+							</div>
+						</div>
+						<div
+							class="select-menu-list"
+							data-filterable-for="filter-field"
+							data-filterable-type="substring"
+						>
+							{children}
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	);
+};
+
+const getSelectedTagFromUrl = () => {
+	const currentUrl = location.href;
+	const index = currentUrl.indexOf('/releases/tag/');
+
+	return index > -1 ?
+		currentUrl
+			.substr(index)
+			.split('/')
+			.pop() :
+		null;
```

Comment:
This function can be replaced with a regex, approximately:

```js
const [tag] = location.href.match(/releases\/tag\/(^\/+)/) || [];
return tag;
```

Reply:
Done. Replaced with `location.href.match(/releases\/tag\/([^>/]*)/)`. Your suggestion was not working.